### DISCUSSION TITLE
Create an `awesome-swc` repo

### DISCUSSION THREAD

**Discussion by chenjiahan** [swc-project/swc#9116] [General]
Considering that there are more and more SWC plugins in the community, it would be great if SWC could have an `awesome-swc` repo and list all the awesome things related to SWC. 😄

**Comment by chenjiahan** [comment]
I searched and found an `awesome-babel-to-swc` repository, but it doesn't look actively maintained:

https://github.com/arnaugomez/awesome-babel-to-swc

**Reply by kdy1** [reply]
It sounds good to me. I made `swc-contrib` org on GitHub and https://github.com/swc-contrib/awesome-swc

**Reply by chenjiahan** [reply]
Awesome 👍 I would like to help add more plugins to the repo

**Reply by kdy1** [reply]
I sent you an invitation to `swc-contrib`
